src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -690,7 +686,7 @@ public sealed override ImmutableArray<ParameterSymbol> Parameters
         internal override bool IsExplicitInterfaceImplementation
             => (_propertyFlags & Flags.IsExplicitInterfaceImplementation) != 0;
 
-        private bool HasGetAccessor => (_propertyFlags & Flags.HasGetAccessor) != 0;
+        internal bool HasGetAccessor => (_propertyFlags & Flags.HasGetAccessor) != 0;
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

There is an existing API to check for a setter. We don't need to expose a new one.